.github/workflows/_reusable_deploy.yml

```
@@ -90,3 +86,71 @@ jobs:
+
+  deploy-fhir-validate-server:
+    name: Deploy FHIR Validate Server
+    concurrency:
+      group: fhir-validate-${{ inputs.deploy_env }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.deploy_env }}
+    defaults:
+      run:
+        working-directory: "./infra"
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)